proxy/coco/src/peer/run_state.rs

```diff
@@ -192,6 +197,23 @@ impl Default for SyncConfig {
     }
 }
 
+/// Set of knobs to alter request behaviour.
+pub struct RequestConfig {
```

Comment:
These poll rates don't seem like a thing that is frequently configured or changed from the default. They are in fact internals and this seem to leak them.